config/sync/migrate_plus.migration.d7_file_private.yml

```diff
@@ -0,0 +1,46 @@
+uuid: ffb73cbc-8902-4da4-aa46-e21a9267e567
+langcode: en
+status: true
+dependencies: {  }
+id: d7_file_private
+class: null
+field_plugin_method: null
+cck_plugin_method: null
+migration_tags:
+  - dept_sites
+migration_group: migrate_group_files
+label: 'Private files'
+source:
+  plugin: d7_file_uuid
+  scheme: private
+  track_changes: true
+  constants:
+    source_base_path: /app/imports/files/private/
+process:
+  filename: filename
+  source_full_path:
+    -
+      plugin: concat
+      delimiter: /
+      source:
+        - constants/source_base_path
+        - filepath
+  uri:
+    plugin: file_copy
+    source:
+      - '@source_full_path'
+      - uri
+    move: false
```

Comment:
Need to copy as otherwise we'd end up re-importing files we've already got, as the nightly sync would re-fetch them and they'd be seen as new things to import.